ts/components/messages/CalendarEventButton.tsx

```diff
@@ -227,14 +233,23 @@ class CalendarEventButton extends React.PureComponent<Props, State> {
   };
 
   // Create an action to add or remove the event
-  private onPressHandler = () => {
+  private onPressHandler = async () => {
     const { calendarEvent, message, preferredCalendar } = this.props;
     const { due_date } = message.content;
 
     if (due_date === undefined) {
       return;
     }
 
+    await requestIOAndroidPermission(
```

Comment:
Since we also read the calendar (to avoid to add the same event twice)
https://github.com/pagopa/io-app/blob/4dd27500525d15da63e33361998289ff210de69b/ts/components/messages/CalendarEventButton.tsx#L192
Maybe should we ask also for READ permission?

I checked the Android code and it asks for both (when we invoke `checkPermission` we don't ask for readonly permission)
![Schermata 2021-11-18 alle 11 20 54](https://user-images.githubusercontent.com/822471/142397700-f9fba0b3-4b7c-4a73-a664-89abd8d4d4ab.png)
h